app/services/foreman/access_permissions.rb

```diff
@@ -347,7 +347,8 @@
                                                :update_multiple_hostgroup, :update_multiple_parameters, :toggle_manage,
                                                :select_multiple_organization, :update_multiple_organization,
                                                :disassociate, :multiple_disassociate, :update_multiple_disassociate,
-                                               :select_multiple_location, :update_multiple_location].push(*ajax_actions),
+                                               :select_multiple_location, :update_multiple_location, :rebuild_config,
+                                               :submit_rebuild_config].push(*ajax_actions),
```

Comment:
There's an inconsistency here between the UI and API - the API permission is on build_hosts, but the UI permission is on edit_hosts.  I'd suggest perhaps moving the UI permission to build_hosts.